lhotse/bin/modes/workflows.py

```diff
@@ -92,3 +93,61 @@ def annotate_with_whisper(
             desc="Recordings",
         ):
             writer.write(cut, flush=True)
+
+
+@workflows.command()
+@click.argument(
+    "in_cuts", type=click.Path(exists=True, dir_okay=False, allow_dash=True)
+)
+@click.argument("out_cuts", type=click.Path(allow_dash=True))
+@click.option(
+    "-n",
+    "--bundle-name",
+    default="WAV2VEC2_ASR_BASE_960H",
+    help="One of Torchaudio pretrained 'bundle' variants (see: https://pytorch.org/audio/stable/pipelines.html)",
```

Comment:
```suggestion
    help="One of torchaudio pretrained 'bundle' variants (see: https://pytorch.org/audio/stable/pipelines.html)",
```

just to be consistent with other usages of `torchaudio` in this file.